static/css/thing.css

```diff
@@ -21,6 +21,23 @@
   background-image: url('../images/unknown-thing.png');
 }
 
+.thing-details-link {
+  box-sizing: border-box;
+  display: block;
+  position: relative;
+  z-index: 10;
+  float: right;
+  background-color: #89b6d6;
+  background-size: 3.2rem;
```

Comment:
Can you make this 2.4rem instead of 3.2rem? The bubble is smaller than the mockups (which it does actually need to be if it's going to line up with the right of the icon), so I think the icon needs to shrink as well. Hopefully this is a big enough touch target.